src/rpc/client/page.ts

```diff
@@ -501,7 +501,17 @@ export class Page extends ChannelOwner<PageChannel, PageInitializer> {
   }
 
   async pdf(options: types.PDFOptions = {}): Promise<Buffer> {
-    const binary = await this._channel.pdf(options);
+    const transportOptions: PDFOptions = options as PDFOptions;
```

Comment:
You have to make a deep copy here to avoid modifying user object.

Reply:
Done